addons/portal/controllers/portal.py

```diff
@@ -264,6 +269,24 @@ def _update_password(self, old, new1, new2):
 
         return {'success': {'password': True}}
 
+    def _revoke_all_devices(self, pwd):
+        if not pwd:
+            return {'errors': {'password': {'password': _("You cannot leave the password field empty.")}}}
+
+        try:
+            request.env['res.users'].change_password(pwd, pwd)
+        except AccessDenied as e:
+            msg = e.args[0]
+            if msg == AccessDenied().args[0]:
+                msg = _('The password you provided is incorrect, accesses were not revoked.')
```

Comment:
```suggestion
                msg = _('Incrorrect password')
```
as the action failed, it's understood nothing happened